cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/program/MessagingProgramStateWriter.java

```diff
@@ -55,15 +50,7 @@ public final class MessagingProgramStateWriter implements ProgramStateWriter {
   private final ProgramStatePublisher programStatePublisher;
 
 
-  @Inject
-  public MessagingProgramStateWriter(CConfiguration cConf, MessagingService messagingService) {
-    this(new MessagingProgramStatePublisher(messagingService,
-                                            NamespaceId.SYSTEM.topic(cConf.get(
-                                              Constants.AppFabric.PROGRAM_STATUS_EVENT_TOPIC)),
-                                            RetryStrategies.fromConfiguration(cConf, "system.program.state.")));
-  }
-
-  @VisibleForTesting
+  @Inject @VisibleForTesting
```

Comment:
nit: we normally have annotations on different lines

Reply:
Done